Mathlib/CategoryTheory/Abelian/GrothendieckAxioms.lean

```diff
@@ -92,6 +88,38 @@ lemma hasExactColimitsOfShape_of_equiv {J J' : Type*} [Category J] [Category J']
   haveI : HasColimitsOfShape J' C := hasColimitsOfShape_of_equivalence e
   ⟨preservesFiniteLimits_of_natIso (Functor.Final.colimIso e.functor)⟩
 
+variable {C} in
+lemma hasExactColimitsOfShape_obj_of_equiv (J : Type*) {D : Type*} [Category J] [Category D]
+    (e : C ≌ D) [HasColimitsOfShape J C] [HasExactColimitsOfShape J C] :
+    haveI : HasColimitsOfShape J D := Adjunction.hasColimitsOfShape_of_equivalence e.inverse
+    HasExactColimitsOfShape J D := by
```

Comment:
This now follows from `HasExactColimitsOfShape.domain_of_functor J e.functor`

Reply:
It almost does, but `domain_of_functor` requires that finite limits exist -- so I think this still has a reason to exist.